src/main/java/frc/robot/subsystems/climb/ClimbSubsystem.java

```diff
@@ -37,6 +39,7 @@ public void initialize() {
 
       climbMotorRight.setIdleMode(IdleMode.kCoast);
       climbMotorLeft.setIdleMode(IdleMode.kCoast);
+      mode = IdleMode.kCoast;
```

Comment:
I know it is unrelated to this PR; but should the climb motors be in Brake mode?